docs/synapse/adminguide.rstorm

```diff
@@ -789,18 +789,16 @@ Display information for the current layer:
 .. storm-cli:: auth.gate.show $lib.layer.get().iden
 
 
-.. _admin_axon_perms:
+.. _admin_cortex_perms:
```

Comment:
there is a link to the `admin_axon_perms` anchor in this guide, do we want to point that to axon info in the devops guide?

Reply:
I think this may have been confusion based on the perms you can assign to service accounts vs users. I'll check on the broken cross ref, but we shouldn't be discussing real service perms here anymore :)

Reply:
removed. the other links in that list are still valid discussion points.